### DISCUSSION TITLE
Unable to Receive Emails on Postal

### DISCUSSION THREAD

**Discussion by mamnoonulhussnain** [postalserver/postal#1547] [Help with using Postal]
Hi! I have configured all the DNS records. 

When I try to send an email from my Gmail to my Postal, I get an error which says error 530 Authentication  required.

How to resolve this please?

**Comment by willpower232** [comment]
do you mean you are trying to send an email to a route in Postal?

**Comment by mamnoonulhussnain** [comment]
I thought postal can receive emails. Just read that it doesn't so I added a route

**Reply by willpower232** [reply]
has that resolved the error you are seeing?

**Comment by fiantinangon** [comment]
I also experienced the same issue @willpower232 

1.  All DNS well configured
3. MX record well configured 

But When I try to send an email from my Gmail to my Postal, I get an error which says error 530 Authentication required.

**Reply by willpower232** [reply]
#1658
